src/core/locator/qgslocator.h

```diff
@@ -171,6 +191,8 @@ class CORE_EXPORT QgsLocator : public QObject
     QList< QgsLocatorFilter * > mFilters;
     QList< QThread * > mActiveThreads;
 
+    QStringList mAutocompleList;
```

Comment:
Should be something like `mAutocompletionList`, or is it intentional?

Reply:
fixed, thanks